site/pages/what-we-do/header-slice/index.js

```diff
@@ -10,13 +10,21 @@ const headerSlice = () => {
           <h1 className={styles.pageHeader}>
             Do the<br />right thing
           </h1>
-          <img alt="" className={styles.arrowDown} src={arrowImg} />
+          <img
+            alt="Arrow down from innovation phrase to delivery phrase."
+            className={styles.arrowDown}
+            src={arrowImg}
+          />
         </div>
         <h3 className={styles.pageSubHeader}>Innovation</h3>
       </div>
       <div className={styles.right}>
         <div className={styles.flexible}>
-          <img alt="" className={styles.arrowUp} src={arrowImg} />
+          <img
+            alt="Arrow back up from delivery phrase to innovation phrase."
```

Comment:
Can we check this alt text with @stevecottle ?

Reply:
@dpiatek That's already on our list :-). @joaquimadraz and I spoke about it yesterday.

Reply:
Ok cool

Reply:
@joaquimadraz @dpiatek What are your thoughts on tagging this up as a whole diagram? For example we put a box around the diagram and have a tag which reads something like 'Cyclic diagram of doing the right thing through innovation and delivery' and ignores the individual components. I'm thinking tagging up the individual components might not be that helpful for those who use screen readers. Also to get a screen reader to focus on each component in a circular fashion might not be possible.

Reply:
@stevecottle 
That makes sense to me. It also reduce the number keystrokes to traverse the page which is good.
I still have to add the attributes for keyboard navigation.

Reply:
@joaquimadraz Great, let's give it a go. Do you need anything from me?

Reply:
I think that might be possible with ARIA label attributes, but maybe a simpler solution would to just have the elements as an ordered list? and then the arrow images can again have `alt=""` as order of things can be followed from HTML?